tests/cc/test_usdt_args.cc

```diff
@@ -113,7 +121,7 @@ TEST_CASE("test usdt argument parsing", "[usdt]") {
     verify_register(parser, 2, 1097);
     verify_register(parser, 4, "gpr[30]", 108);
     verify_register(parser, -2, "gpr[31]", -4);
-#elif defined(__x86_64__)
```

Comment:
why remove defined(__x86_64__) ?
Other than this nit it all looks good to me.
cc @goldshtn

Reply:
This is for consistency. In all other places, we have x86_64 as the default. So I changed a couple of places here to have x86_64 as the default. 
 
The x86_64 as the default in the core source files makes build easier for not-fully-supported architectures. Although some tests/tools may not work as expected, at least build can go through without change.